data/data/rhcos.json

```diff
@@ -1,114 +1,134 @@
 {
     "amis": {
         "ap-northeast-1": {
-            "hvm": "ami-010b504a77f140648"
+            "hvm": "ami-047db84763a6bd5ee"
         },
         "ap-northeast-2": {
-            "hvm": "ami-08a55600a8077584d"
+            "hvm": "ami-0edd8208e96e52e5d"
         },
         "ap-south-1": {
-            "hvm": "ami-05aea9fb95d4d0aca"
+            "hvm": "ami-0fca367bf64bd7796"
         },
         "ap-southeast-1": {
-            "hvm": "ami-038b194e27fc2ec49"
+            "hvm": "ami-0f3d5ee8e42749b11"
         },
         "ap-southeast-2": {
-            "hvm": "ami-0245225726ea46566"
+            "hvm": "ami-08ce4dc926f2109e4"
         },
         "ca-central-1": {
-            "hvm": "ami-0c6791e3ed449e0ea"
+            "hvm": "ami-07d2b07b9221d1490"
         },
         "eu-central-1": {
-            "hvm": "ami-03b60221c0df162fb"
+            "hvm": "ami-05ab0f5bf64ea1b23"
         },
         "eu-north-1": {
-            "hvm": "ami-075ca0ed672981968"
+            "hvm": "ami-0fd06928d41d7c9fa"
         },
         "eu-west-1": {
-            "hvm": "ami-0a660bc79a911027a"
+            "hvm": "ami-0e0ceb0e2273200c1"
         },
         "eu-west-2": {
-            "hvm": "ami-0bc1a9a6387604e16"
+            "hvm": "ami-0c150a875711ef355"
         },
         "eu-west-3": {
-            "hvm": "ami-073a51d01475014a3"
+            "hvm": "ami-0927269587a0b1035"
         },
         "sa-east-1": {
-            "hvm": "ami-0e33fcaf038fed396"
+            "hvm": "ami-016704e63067d0eae"
         },
         "us-east-1": {
-            "hvm": "ami-02feada323df64a90"
+            "hvm": "ami-028d386c3abb95117"
         },
         "us-east-2": {
-            "hvm": "ami-037bdb4a9a790a950"
+            "hvm": "ami-02b7ff91dfee1781c"
         },
         "us-west-1": {
-            "hvm": "ami-009497e98fe8153e4"
+            "hvm": "ami-0ecbe747de22df687"
         },
         "us-west-2": {
-            "hvm": "ami-0a251abf272da1b9e"
+            "hvm": "ami-08104ac81b604e970"
         }
     },
-    "baseURI": "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.2/420.8.20190708.2/",
-    "buildid": "420.8.20190708.2",
+    "azure": {
+        "image": "rhcos-42.80.20190724.3.vhd",
+        "url": "https://rhcos.blob.core.windows.net/imagebucket/rhcos-42.80.20190724.3.vhd"
+    },
+    "baseURI": "https://releases-rhcos-art.cloud.privileged.psi.redhat.com/storage/releases/rhcos-4.2/42.80.20190724.3/",
```

Comment:
this is private url. needs to be the public one.

Reply:
Indeed. Remove private and use public.